templates/home.tmpl

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta
+            name="description"
+            content="A better way to book club" />
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1" />
+
+        <title>CerealNotes</title>
+
+        <link href="//cdn.muicss.com/mui-0.9.36/css/mui.min.css" rel="stylesheet" type="text/css" />
+
+        <script src="//cdn.muicss.com/mui-0.9.36/js/mui.min.js"></script>
+        <script src="//ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+        <script src="/static/js/home.js"></script>
+    </head>
```

Comment:
We'll deal with DRYing up this stuff in a later PR

Reply:
if that means less copy pasta-ing i'm all game. Should we make a task for it?

Reply:
created https://trello.com/c/EqKke9zl